src/Orleans/CodeGeneration/TypeUtils.cs

```diff
@@ -441,5 +461,343 @@ public static bool TypeHasAttribute(Type type, Type attribType)
             return CustomAttributeData.GetCustomAttributes(type).Any(
                     attrib => attribType.IsAssignableFrom(attrib.AttributeType));
         }
+
+        /// <summary>
+        /// Returns the grain methods from the provided <paramref name="interfaceType"/>.
+        /// </summary>
+        /// <param name="interfaceType">The interface.</param>
+        /// <returns>The grain methods on the provided <paramref name="interfaceType"/>.</returns>
+        public static Dictionary<int, MethodInfo> GetGrainMethods(Type interfaceType)
```

Comment:
Thanks for pointing that out, I've replaced it with the existing implementation.